src/libs/filter.ts

```diff
@@ -0,0 +1,27 @@
+// stackoverflow.com/questions/990904/remove-accents-diacritics-in-a-string-in-javascript/37511463#37511463
+function toNFD (s: string): string {
+  return s.normalize('NFD').replace(/[\u0300-\u036f]/g, '')
+}
+
+export function Assert (obj: any, query: string, field: string, ...fields: Array<string>): boolean {
+  if (!obj) {
+    return false
+  }
+
+  fields.push(field)
+
+  query = toNFD(query.trim().toLowerCase())
+
+  for (const f of fields) {
```

Comment:
Could we replace lines 15-24 with

`fields.some(f => toNFD((obj[f]  || '').trim().toLowerCase()).includes(query)`
Otherwise LGTM